### PR TITLE
Add --no-template option for make:controller command

### PR DESCRIPTION
**Pull request by welcoMattic**
I had several times the case in which I did not necessarily need a template when I generated a controller. So I'm trying to add an option `--no-template` to force avoid template generation.

I think that I will need help on tests writing ;)

### CODE REVIEW

File: src/Maker/MakeController.php [hunk 1]
```diff
@@ -38,6 +39,7 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
         $command
             ->setDescription('Creates a new controller class')
             ->addArgument('controller-class', InputArgument::OPTIONAL, sprintf('Choose a name for your controller class (e.g. <fg=yellow>%sController</>)', Str::asClassName(Str::getRandomTerm())))
+            ->addOption('--no-template', '-n', InputOption::VALUE_OPTIONAL)
```
Comment (codedmonkey) [inline src/Maker/MakeController.php]: I think you mean `InputOption::VALUE_NONE`

**Comment by weaverryan** [comment]
Hey @welcoMattic!

In general, I want to be careful about adding customizations like this. But, this one seems reasonable.

It looks like some of the existing controller tests broke. Can you check those out? I'm not sure what's going on - I'd try to run those test cases locally if you can.

Cheers!

**Comment by welcoMattic** [comment]
Yes, existing tests broke. It seems to be because of `addOption` method, but it looks weird that this method broke tests 🤔 

If you can run those tests locally to help, that would be nice 😉 

Thanks

**Comment by weaverryan** [comment]
Hey! Would you mind running the tests locally? I’m happy to review or debug (especially if the test suite doesn’t has any issues running on your machine), but I’d prefer you to finish this feature if you’d like it :)

**Comment by weaverryan** [comment]
Sorry for the delay, but this was an excellent PR - very clean. Thank you @welcoMattic!

**Pull request merged**